documented label cluster-service (#48988)
kubernetes/website: content/en/docs/reference/labels-annotations-taints/_index.md

Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassName`.
{{< /note >}}

> ### kubernetes.io/cluster-service (deprecated)

Reviewer: ```suggestion
### kubernetes.io/cluster-service (deprecated) {#kubernetes-io-cluster-service}
```

Author: made changes as suggested

Type: Label

Example: kubernetes.io/cluster-service: "true"

Used on: Service

> Addon manager deprecates this label for usage in addon manager. It is not deprecated in general.
> kubectl cluster-info depends on it and this annotation appear there.

Reviewer: ```suggestion
This label indicates that the Service provides a service to the cluster, if the value is set to true.
When you run `kubectl cluster-info`, the tool queries for Services with this label set to true.

However, setting this label on any Service is deprecated.
```
I wouldn't mention addon manager here.

Author: made changes as suggested

Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassName`.
{{< /note >}}

### kubernetes.io/cluster-service (deprecated) {#kubernetes-io-cluster-service}

Type: Label

> Example: kubernetes.io/cluster-service: "true"

Reviewer 2: _Minor non-blocking nit_: Adjust the formatting of the example block to match the style of all the other examples on this [page](https://kubernetes.io/docs/reference/labels-annotations-taints/).

```suggestion
Example: `kubernetes.io/cluster-service: "true"`
```

Author: @dipesh-rawat incorporated suggestion.
